src/ga4gh/vrs/extras/translator.py

```diff
@@ -410,10 +438,44 @@ def _to_spdi(
         """
         sequence = f"ga4gh:{vo.location.get_refget_accession()}"
         aliases = self.data_proxy.translate_sequence_identifier(sequence, namespace)
-        aliases = [a.split(":")[1] for a in aliases]
+        aliases = [a.split(":")[1] for a in aliases]  # TODO why split here?
```

Comment:
An example of an alias that is returned is `refseq:NC_000021.9`, in the final SPDI expression we don't care about the curie prefix (`refseq:`)

```suggestion
        aliases = [a.split(":")[1] for a in aliases]
```